src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1548,10 +1548,40 @@ public void Copy()
         }
 
         /// <summary>
-        ///     Paste ISelectable objects from the clipboard to the workspace.
+        ///     Paste ISelectable objects from the clipboard to the workspace 
+        /// so that the nodes appear in their original location with a slight offset
         /// </summary>
         public void Paste()
         {
+            // Provide a small offset when pasting so duplicate pastes aren't directly on top of each other
+            CurrentWorkspace.IncrementPasteOffset();
+
+            var locatebleModels = ClipBoard.Where(model => model is NoteModel || model is NodeModel);
```

Comment:
Annotations did not participate in computing `minXY`, `maxXY`. So I have not included them here